src/infrastructure/network/switches.md

```diff
@@ -32,24 +33,26 @@ We've configured groups of ports to map to certain [VLANs](../network/vlans.md).
 | Ports | VID | Name       | Speed |
 |-------|-----|------------|-------|
 | 0-7   | 3   | cosi\_priv | 10 G  |
```

Comment:
iirc 0-7 should be cosi pub now

Reply:
They are on FCOLO, on FHILL they are all on private right now

Reply:
I thought we left some in case of future need to have the public vlan at the hill for things like VMs or other services. Though, if we are placing a vm machine in colo, this would take care of the big reason for this decision.

If this is what the labs decided on, then the vlan id should still be changed to 2 anyways.

Reply:
That would be ideal, unfortunately we never set up our connection between the hill campus and COLO as a trunk, so right now it's just private coming back to campus. Thank you for pointing out the VID, I'll get that corrected